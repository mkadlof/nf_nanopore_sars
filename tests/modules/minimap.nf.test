nextflow_process {

    name "Test Process minimap"
    script "modules/minimap.nf"
    process "minimap"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                input[0] = file("${projectDir}/testing_data/ESIB_EQA_2024_SARS1_01.fastq.gz")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
