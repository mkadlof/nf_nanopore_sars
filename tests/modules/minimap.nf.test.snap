{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        "ESIB_EQA_2024_SARS1_01",
                        "sorted.bam:md5,1c3483a21382d0c6fb3c80a656c68f37",
                        "sorted.bam.bai:md5,dc6f03362b2b0952d10d42862b1218f6"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-05-16T08:21:18.807542168"
    }
}